Simonyi/Troubleshooting/MTCS/MTM2/M2-recovery.rst

```diff
@@ -0,0 +1,338 @@
+
+         .. figure:: _static/MTM2-recovery-restart-control-system.png
+            :width: 600
+
+            Restarting MTM2 control system
+
+    .. _M2-recovery-Reset-the-M2-interlock-signal: 
+
+#. `Reset the M2 interlock signal`_ in GIS main cabinet on level 2, even if the state is "OK".
+     .. Important::
+
+        Note that all status boxes for the M2 actuator will appear green. This indicates the status of the relay that enables power to the systems, not the status of M2 itself. Therefore, **after an interlock or power cycling**, it is necessary to press the :guilabel:`RESET` button.
+
+#. Use python EUI/GUI to change MTM2 to *closed-loop* state:
+     #. Open the **MTM2 EUI**. Follow instructions :ref:`to access the MTM2 EUI <EUI-Access-Accessing-M2-Camera-Hexapods-and-Camera-Rotator-EUIs>`.
+
+     #. Establish **local control** by pressing :guilabel:`connect`, then :guilabel:`local`. 
+         Note that :guilabel:`local` may be greyed out after connecting, this is normal.
+
+         .. figure:: _static/MTM2-recovery-GUI-open-connect.png
+            :width: 600
+            :align: center
+
+            MTM2 GUI open and connect
+
+     #. Pull up the **Overview widget** by double-clicking on :guilabel:`Overview` in the list at the bottom of the EUI.
+         .. figure:: _static/MTM2-recovery-GUI-overview.png
+            :width: 600
+            :align: center
+
+            MTM2 GUI Overview
+
+         #. Check the **Enabled Faults Mask**. 
+             It should **not be 0**. If it is, repeat `Reset the M2 interlock signal`_.
+             
+             .. note::
+                
+                It is ok if the ``isInterlockEngaged`` indicator is red.  
+
+                .. the original said isInterlockEnabled
+
+         #. Look at **Alarms/Warnings** widget to see active alarms (red) or warnings (yellow). 
+             If active, reset them with :guilabel:`Reset All Items`. 
+        
+             *Make sure you have removed the fault condition*.
+
+             .. figure:: _static/MTM2-recovery-GUI_alarms-warnings.png
+                :width: 100%
+                :align: center
+
+                GUI Alarms and warnings widget
+
+             If **Reset All Items** does not work, you maybe have to :ref:`power cycle M2 cabinet <M2-Non-standard-Procedures-Power-cycle-MTM2-cabinet>`. 
+             *Only do this if there are no other options!*
+
+     #. Switch to :guilabel:`Diagnostic` mode. Be patient; this may take some time.
+
+     #. Switch to  :guilabel:`Enabled` mode. This may take up to 2 minutes. If this step fails, you may have to repeat `Reset the M2 interlock signal`_ instructions.
+
+     #. :guilabel:`Enter closed-loop control`.
+
+#. Return to **Standby mode** in the EUI to close the GUI by **pressing the following buttons**:
+     #. :guilabel:`Enter open-loop control`.
+
+     #. :guilabel:`Diagnostic` mode, this usually takes ~30s.
+
+     #. :guilabel:`Standby` mode, this usually takes ~30s.
```

Comment:
David S. pointed out to me the other day that there is an additional step needed here following Standby.

#. guilabel:`Remote` mode, to allow CSC control of M2.